provisioner/clusterpy.go

```diff
@@ -258,9 +262,10 @@ func (p *clusterpyProvisioner) provision(
 	}
 
 	values := map[string]interface{}{
-		subnetsValueKey:             azInfo.SubnetsByAZ(),
-		availabilityZonesValueKey:   azInfo.AvailabilityZones(),
-		subnetIPV6CIDRsKey:          strings.Join(azInfo.SubnetIPv6CIDRs(), ","),
+		subnetsValueKey:             azInfoWorkers.SubnetsByAZ(),
+		availabilityZonesValueKey:   azInfoWorkers.AvailabilityZones(),
+		subnetIPV6CIDRsKey:          strings.Join(azInfoWorkers.SubnetIPv6CIDRs(), ","),
+		"lb_subnets":                azInfoLBs.SubnetsByAZ(),
```

Comment:
This is new, do we need in a similar change in provisionNodePool ?

Reply:
I guess this is handled by `azInfo: azInfoWorkers,`

Reply:
This will be used like this: https://github.com/zalando-incubator/kubernetes-on-aws/pull/8440